validator/build.py

```diff
@@ -74,7 +74,7 @@ def CheckPrereqs():
   try:
     libprotoc_version = subprocess.check_output(['protoc', '--version'])
   except (subprocess.CalledProcessError, OSError):
-    Die('Protobuf compiler not found. Try "apt-get install protobuf-compiler".')
+    Die('Protobuf compiler not found. Follow the install instructions at https://github.com/ampproject/amphtml/blob/master/validator/README.md#installation.')
```

Comment:
I think leaving the specific `Try "apt-get install foo"` is useful, while _adding_ the link to the messages.

Reply:
My fear with that is that it would make a long line longer, and provide a red herring instruction to contributors not on Debian-based Linux distros.